torch/csrc/distributed/c10d/TCPStoreLibUvBackend.cpp

```diff
@@ -1012,7 +1012,7 @@ void LibUVStoreDaemon::init(const TCPStoreOptions& opts) {
     try {
       tcpServer = UvTcpServer::makeWithPort(&loop, opts.port, /*useIpv6=*/true);
     } catch (std::exception& ex) {
-      C10D_INFO(
+      C10D_WARNING(
           "Failed to bind to ipv6 address, trying ipv4. Error: {}", ex.what());
       tcpServer =
           UvTcpServer::makeWithPort(&loop, opts.port, /*useIpv6=*/false);
```

Comment:
should this "trying ipv4" also be in a `try/catch`?

Reply:
nah -- the ipv4 one will throw an exception and fail if it can't bind anything